tests/integration/components/o-s-s/power-select-test.ts

```diff
@@ -242,9 +243,14 @@ module('Integration | Component | o-s-s/power-select', function (hooks) {
         </div>
       `);
 
-      await scrollTo('.upf-infinite-select__items-container', 0, 100);
+      await scrollTo(
+        '.upf-infinite-select__items-container',
+        0,
+        document.querySelector('.upf-infinite-select__items-container')!.scrollHeight -
+          document.querySelector('.upf-infinite-select__items-container')!.clientHeight
+      );
 
-      assert.ok(this.onBottomReached.calledOnce);
```

Comment:
`calledOnce` should've worked, if it didn't it means that it is called mutliple times, which is weird no ? 🤔

Reply:
The error maybe come from Infinite-select?